mokapot/confidence.py

```diff
@@ -86,14 +86,9 @@ def __init__(
         prefixes=None,
     ):
         """Initialize a GroupedConfidence object"""
-        data = read_file(
-            psms.filename,
-            use_cols=list(psms.feature_columns) + list(psms.metadata_columns),
-        )
+        data = read_file(psms.filename, use_cols=list(psms.columns))
         self.group_column = psms.group_column
         psms.group_column = None
-        scores = scores * (desc * 2 - 1)
```

Comment:
why is this line not needed anymore?

Reply:
that was removed in the original repo from mokapot and forgot to delete it when we did the sync